composer.json

```diff
@@ -20,6 +20,7 @@
         "symfony/asset": "^2.7 || ^3.1",
         "symfony/templating": "^2.7 || ^3.1",
         "symfony/translation": "^2.7 || ^3.1",
+        "pagerfanta/pagerfanta": "^1.0.5",
```

Comment:
this is not a hard dependency at all. we don't use the persistence stuff is optional and we don't use it for example. so we also do not need this pagination, do we?

Reply:
Since 5.x it is used by default. Here:

* https://github.com/FriendsOfSymfony/FOSElasticaBundle/blob/master/Doctrine/MongoDBPagerProvider.php
* https://github.com/FriendsOfSymfony/FOSElasticaBundle/blob/master/Doctrine/ORMPagerProvider.php
* https://github.com/FriendsOfSymfony/FOSElasticaBundle/blob/master/Provider/PagerfantaPager.php

Though FOSElasticaBundle relies on its own `PagerInterface` so one can simply use any other pager implementations or their own.

Reply:
It is only used if the persistence configuration of a type is enabled. So it's optional.